src/main/webapp/app/entities/accessibility-for-disabilities/accessibility-for-disabilities.reducer.ts

```diff
@@ -101,8 +101,8 @@ const apiUrl = 'api/accessibility-for-disabilities';
 
 // Actions
 
-export const getEntities: ICrudGetAllAction<IAccessibilityForDisabilities> = (page, size, sort) => {
-  const requestUrl = `${apiUrl}${sort ? `?page=${page}&size=${size}&sort=${sort}` : ''}`;
+export const getEntities: ICrudGetAllAction<IAccessibilityForDisabilities> = (page, size, sort, itemsPerPage) => {
```

Comment:
What is the difference between size and itemsPerPage? Seems like you are passing twice the same values here https://github.com/benetech/ServiceNet/pull/635/files#diff-02bac1b50216c64d8296a4618de767a9R61

Reply:
I still don't understand why you are sending same param twice to the backend